src/SignalR/common/Protocols.Json/ref/Microsoft.AspNetCore.SignalR.Protocols.Json.netcoreapp3.0.cs

```diff
@@ -18,7 +18,6 @@ public sealed partial class JsonHubProtocol : Microsoft.AspNetCore.SignalR.Proto
     {
         public JsonHubProtocol() { }
         public JsonHubProtocol(Microsoft.Extensions.Options.IOptions<Microsoft.AspNetCore.SignalR.JsonHubProtocolOptions> options) { }
-        public int MinorVersion { get { throw null; } }
```

Comment:
This was new in 3.0, right?

Reply:
Yup. Here's the 2.2 branch of the JsonHubProtocol https://github.com/aspnet/AspNetCore/blob/release/2.2/src/SignalR/common/Protocols.Json/src/Protocol/JsonHubProtocol.cs
No reference to MinorVersion